Assets/MixedRealityToolkit.Extensions/Webrtc/Scripts/WebrtcUIControls.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Webrtc
+{
+    /// <summary>
+    /// This SHOULD NOT BE USED FOR PRODUCTION.
+    /// 
```

Comment:
what does this mean?

Reply:
this actually should be removed (deprecated that sample) in favor of clearer WebrtcSignalControls (more production applicable, extendable).